tensorboard/components/tf_storage/storage.ts

```diff
@@ -218,7 +218,7 @@ namespace tf_storage {
     return {get, set, getInitializer, getObserver, disposeBinding};
   }
 
-  export function migrateLegacyURLScheme() {
```

Comment:
Any reason we have to make the name change?  Also, you will need to update tf-tensorboard then.

Reply:
Reverted back to the old name, and added a comment in tf-tensorboard to explain that order matters.